native/packages/react-native-bpk-component-input/src/BpkInput.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  TextInput,
+  Image,
+  Platform,
+  StyleSheet,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const IOS_TOKENS = require('bpk-tokens/tokens/ios/base.react.native.common.js');
+const ANDROID_TOKENS = require('bpk-tokens/tokens/android/base.react.native.common.js');
+
+const tokens = Platform.select({
+  ios: () => IOS_TOKENS,
+  android: () => ANDROID_TOKENS,
+})();
+
+const {
+  colorGray100,
+  colorGray300,
+  colorGray700,
+  colorGreen500,
+  colorRed500,
+  spacingMd,
+  spacingSm,
+  textLgFontSize,
+  textLgFontWeight,
+  textXsFontSize,
+  textXsFontWeight,
+} = tokens;
+
+const styles = StyleSheet.create({
+  input: {
+    borderColor: colorGray100,
+    borderWidth: 1,
+    borderRadius: spacingSm,
+    padding: spacingSm * 3,
+    flexDirection: 'row',
+    height: 48,
```

Comment:
this can be a multiple of a spacing token, no?

Reply:
Good shout :)